fontforgeexe/collabclientui.c

```diff
@@ -471,30 +471,31 @@ gpointer getSessionCallbacksObject()
 void collabclient_notifySessionJoining( cloneclient_t *cc, FontViewBase* fv )
 {
 #ifdef BUILD_COLLAB
-    
-    printf("AAA collabclient_notifySessionJoining() fv:%p\n", fv);
-    g_signal_emit_by_name( getSessionCallbacksObject(), "joining", fv );
-    
+    if (cc) {
```

Comment:
* `Collaborate->Start session`
* `Collaborate->Connect to session...` (Choose the only option)
* In the original window: `Collaborate->Close local server`
* In the original window: `Collaborate->Disconnect`